### ISSUE TITLE
NVDA does not read the entire contents of web pages

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6405]
When reading a web page with chrome reading with the arrows below NVDA reads a passage, and soon after returns to read a passage already read previously.
For example, open a call on GitHub, read the contents of the call with the arrows below.
NVDA will read a text excerpt and soon after return some Assima lines and will continue indefinitely lendo.O NVDA will read the same passage.
This occurs randomly.
Using NVDA master 13608 and chrome 53.

**Comment by jcsteh** [comment]
Technical: This can be reproduced in GitHub issues with long comments. As you read the comment, the focus is on the link for the author. NVDA scrolls to the cursor each time you move. Eventually, you'll hit something which causes the focus to scroll off the screen. At this point, Chrome kills the accessible for the focus and creates a new one. It then has to fire a new focus event on it, thus throwing NVDA back to the link.

We've seen similar issues in Firefox, but this particular case doesn't seem to affect Firefox. I've also been unable to isolate this behaviour into a small test case.

**Comment by jcsteh** [comment]
Filed [Chromium issue 650124](https://bugs.chromium.org/p/chromium/issues/detail?id=650124).

**Comment by fernando-jose-silva** [comment]
Thank you very much.

**Comment by jcsteh** [comment]
Triage: P2 because this makes some pages unusable (though we don't know how many pages at this stage). We're hoping for a fix from Google.

Technical: Failing a fix from Google, we might be able to work around this as part of #2039. The idea would be to ignore focus changes for a short time after a scrollTo call.

**Comment by Adriani90** [comment]
this is related to #8999.

**Comment by Adriani90** [comment]
I am closing this one in favor of #8999 because the example provided there can be reproduced reliably both in Chrome and Firefox.
